appveyor.yml

```diff
@@ -0,0 +1,54 @@
+version: 0.1.{build}
+image: Visual Studio 2017
+environment:
+  VCPKGRS_DYNAMIC: 1
+  RUST_VERSION: stable
+  LLVM_VERSION: 6.0.1
+  matrix:
+  - TARGET: x86_64-pc-windows-msvc
+    VCPKG_TARGET: x64-windows
+    LLVM_TARGET: win64
+  - TARGET: i686-pc-windows-msvc
+    VCPKG_TARGET: x86-windows
+    LLVM_TARGET: win32
+  - TARGET: x86_64-pc-windows-msvc
+    VCPKG_TARGET: x64-windows
+    RUST_VERSION: beta
+    LLVM_TARGET: win64
+  - TARGET: i686-pc-windows-msvc
+    VCPKG_TARGET: x86-windows
+    RUST_VERSION: beta
+    LLVM_TARGET: win32
+  - TARGET: x86_64-pc-windows-msvc
+    VCPKG_TARGET: x64-windows
+    RUST_VERSION: nightly
+    LLVM_TARGET: win64
+  - TARGET: i686-pc-windows-msvc
+    VCPKG_TARGET: x86-windows
+    RUST_VERSION: nightly
+    LLVM_TARGET: win32
+install:
+- cmd: >-
+    curl -sSf -o rustup-init.exe https://win.rustup.rs/
+    
+    curl -sSf http://releases.llvm.org/%LLVM_VERSION%/LLVM-%LLVM_VERSION%-%LLVM_TARGET%.exe -o LLVM.exe
+    
+    7z x LLVM.exe -oC:\LLVM
+    
+    set LIBCLANG_PATH=C:\LLVM\bin
```

Comment:
Ah, having looked more carefully this seems to be where the issue lies.

Appveyor treats lines in these blocks as continutions unless there's a blank line between each (as above). This seems to have been lost in the lines below.